apps/user/src/queue-listener/queue-listener.spec.int.ts

```diff
@@ -0,0 +1,6 @@
+const queueResponse = {

```

Comment:
If this is data for mocking, you can use a `json` file for this, this is more of a convention.